apps/src/applab/applab.js

```diff
@@ -699,8 +703,8 @@ Applab.onMouseMoveDebugResizeBar = function (event) {
   //  we explicitly set bottom/height styles on the elements above, we need to
   //  do the same for these styles as well)
 
-  codeTextbox.style.minHeight = 'calc(50% - 21px)';
-  debugArea.style.maxHeight = 'calc(50% - 21px)';
+  codeTextbox.style.minHeight = 'calc(50% - 16px)';
```

Comment:
Does [this change](https://github.com/code-dot-org/code-dot-org/commit/a9df1548eabcb2e49a2f347515577ad23859ef1e) clear things up?